ethportal-api/src/types/consensus/execution_payload.rs

```diff
@@ -18,9 +19,13 @@ pub struct ExecutionPayload {
     pub receipts_root: H256,
     pub logs_bloom: Bloom,
     pub prev_randao: H256, // 'difficulty' in the yellow paper
+    #[serde(deserialize_with = "as_u64")]
```

Comment:
I don't think this needs to be in the code, but can you give a brief explanation here of why we need this new library to parse u64?

Reply:
> I don't think this needs to be in the code, but can you give a brief explanation here of why we need this new library to parse u64?

This allows us to deserialize the fields, `block_number` for example, from string, not only from a number. This is required because we receive the data from pandaops as JSON strings.